src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/global_navitagtion/NotificationsComponentObject.java

```diff
@@ -82,19 +81,19 @@ private void waitForNotificationsMessagesToLoad() {
   public void showNotifications() {
     waitForNotificationsLoaded();
     openNotifications();
-    PageObjectLogging.log("#WallNotifications li ul.subnav", "show notifications", true);
+    LOG.success("#WallNotifications li ul.subnav", "show notifications");
   }
 
   /**
    * click notifications bubble
    *
    * @todo: is this needed? the notifications expand on mouse hover so we should use the
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)